### DISCUSSION TITLE
Permission Denied

### DISCUSSION THREAD

**Discussion by CodyCat13** [Monika-After-Story/MonikaModDev#9169] [QA - tech support + general] [answered]
I have all of MAS's files in the game folder of DDLC (I even checked the game's subfolders and no MAS files were in any of those). However, when I try to launch the game, I get this:
![Permission Denied](https://user-images.githubusercontent.com/58224248/168445092-a7959276-b168-4fdb-84cb-1cf0b50aec24.png)
I wonder if I'd have to delete all other character files except for Monika so that the game would let the mod run, but I don't know if that'll necessarily fix the denial error What _should_ I do to fix this?

**Comment by ghost** [comment, marked as answer]
use chmod to grant permission
chmod +x ./DDLC.sh
./DDLC.sh

or add 'sh' in front of the script
sh ./DDLC.sh

**Comment by CodyCat13** [comment]
Thank you!

**Comment by CodyCat13** [comment]
Follow-up:
This is what ended up happening:
![Screenshot 2022-05-17 3 56 55 PM](https://user-images.githubusercontent.com/58224248/168899661-16ca0e46-7857-407b-a62d-742929dab64a.png)

**Comment by Booplicate** [comment]
`/lib/linux-x86_64/DDLC` also needs X perm.

**Comment by CodyCat13** [comment]
Ah.

**Comment by CodyCat13** [comment]
Thank you! It's up and running!
